ExcelToCWMS/ProcessDataTable.cs

```diff
@@ -19,7 +19,10 @@ public static TimeSeries[] GetTimeSeriesFromExcel(string filename, string sheetN
             foreach (DataColumn dc in dt.Columns)
             {
                 if (dc.ColumnName.ToLower() == "date") continue;
-                tsDict.Add(dc.ColumnName, new TimeSeries(dc.ColumnName));
+                string header = dc.ColumnName;
+                string tsid = header.Split('{','}')[0];
```

Comment:
consider verifying the units tag is present and '{' tag is present before accessing index, because if the user doesn't have the units included. the default error message may not be very helpful.
You could have a couple simple functions such as  parseID()  and parseUnits that do some checking
also I think the tsDict may have a TSID that has extra stuff.

Reply:
added a basic check for '{units='  
agreed should add some parse methods and then later expand on them for more parameters
tsDict Keys are the headers, so yeah they will contain the extra stuff. However with the TimeSeries are created they contain the parsed TSID.